lib/board.ex

```diff
@@ -1,57 +1,47 @@
 defmodule Board do
 
-  @markers ["X","O"]
-
-  def create(board_size) do
-    {:x, [["1","2","3"],["4","5","6"],["7","8","9"]], :first_turn}
+  def create do
+    [["1","2","3"],["4","5","6"],["7","8","9"]]
   end
 
-  def place_marker(selected_tile, game_status) do
-    marker = elem(game_status, 0)
-    elem(game_status, 1)
-    |> check_tile_not_taken(selected_tile, marker)
-  end
+  def place_marker(selected_tile, board, player_symbol) do
+    marker_string = Markers.get_player_marker(player_symbol)
+    index_selection = zero_index_selection(selected_tile)
```

Comment:
I would ask if it is really the responsibility of the Board to make the index zero based? What if the display of the grid changed and didn't display numbers 1-9?